### PR TITLE
DELETE: Opponent vehicle

### PR DESCRIPTION
**Pull request by jenniesparza**

### CODE REVIEW

File: AirSim/Unreal/Environments/Blocks/Config/DefaultInput.ini
Comment (Cybower) [inline AirSim/Unreal/Environments/Blocks/Config/DefaultInput.ini:1]: Was this change intentional? Not sure that we need to implement the enemy vehicle controls

**Comment by Cybower** [comment]
To fix the merge conflict, add the Blueprints to `Blueprint List.md`, then 

`git checkout origin/main -- Airsim/Unreal/Environments/Blocks/Content/Maps/FlyingExample.umap`
`git add .`
`git commit -m "Message Here"`
`git push origin`

**Comment by jenniesparza** [comment]
Can we cancel this Pull Request? I created a new branch that contains the stuff I need from here + the path

**Comment by Cybower** [comment]
Closing Pull PR.
